source/sphinx_directives/command_block.py

```diff
@@ -185,6 +187,14 @@ def _get_output_links(self, output_paths, name):
                 content.append('')
         return content
 
+    def _parse_multiline_commands(self, previous, next):
+        if previous and previous[-1].endswith('\\'):
+            previous[-1].replace('\\', '')
```

Comment:
It's unlikely to happen, but just doing a blanket replace might catch things that are escaped. I would refine this to only strip the last character.